### DISCUSSION TITLE
Pi-hole for Android APK Installer v.3.0.2

### DISCUSSION THREAD

**Discussion by DesktopECHO** [DesktopECHO/Pi-hole-for-Android#34] [Announcements]
### Note to users of previous builds:
- Uninstall any previous versions of Linux Depoy or Pi Deploy and reboot your device.
- Failure to heed this advice will cause issues!

### Installation:
- Download and Install the latest [Pi Deploy APK](https://github.com/DesktopECHO/Pi-hole-for-Android/releases/latest/download/pideploy.apk)
- Tap the main menu (**Three dots** at the top right of screen)
- Tap **New Deployment**
- In a few minutes, the [Raspbian Image](https://github.com/DesktopECHO/linuxdeploy-images/releases/latest/download/raspbian.tgz) will be downloaded and installed on your device.
- When deployment is complete, tap **[  ▷ START ]**  to launch the instance.
- The instance will provide you with a password for Pi-hole admin, SSH and RDP access (Username: _android_, see screenshot below)
- **Note**: The password appears only once when the image is deployed, make sure you record this information.
- **Hint**:  The password text can be copy/pasted to other Android apps (like Remote Desktop) 
- _(Optional)_ To install PIXEL Desktop run `pideploy-gui-install` and restart the instance.  Next time you login via RDP you will have a full desktop session with audio support.

![image](https://user-images.githubusercontent.com/33142753/196851777-e46b145f-4c99-4b6f-9add-ed2f009dae4b.png)

![Screenshot_20221020-013907_Bromite](https://user-images.githubusercontent.com/33142753/196860440-1723d8c2-09b0-460b-901e-260b5485d554.png)

![image](https://user-images.githubusercontent.com/33142753/196856874-72c307e3-2227-4ef1-a7b5-401e745f918f.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/DesktopECHO/Pi-hole-for-Android/releases/tag/v3.0.2'>Pi-hole for Android APK Installer v.3.0.2</a>.</em>

**Comment by Bruzzler0** [comment]
I am not getting anywhere with the installation.

wget: note: TLS certificate validation not
implemented

**Comment by DesktopECHO** [comment]
Just wait a bit, it’s downloading a 300MB disk image.

**Reply by Bruzzler0** [reply]
I'll give it a try. Thanks for the quick reply

**Comment by Derina12** [comment]
Please it says - Requires superuser privileges (root)
Thank you

**Reply by DesktopECHO** [reply]
It means your device is not [rooted.](https://en.wikipedia.org/wiki/Rooting_(Android))

**Reply by Derina12** [reply]
Yes my device is not rooted

**Reply by DesktopECHO** [reply]
> Yes my device is not rooted

It’s very clearly stated that your device needs to be rooted for this to work.

**Reply by Derina12** [reply]
So What can i do Sir ?

**Reply by LeBazarDeBryan** [reply]
> So What can i do Sir ?

Root your device

**Comment by eeb20791** [comment]
Is this going to be safe for my phone even though it says it can harm my device?

**Reply by LeBazarDeBryan** [reply]
Yes it's safe, I tested the apk.

**Comment by warmike01** [comment]
Is this good for Android 4.x?

**Reply by DesktopECHO** [reply]
You want the legacy branch, but I've not worked with it in a while and upstream dropped support for CentOS 7.

[https://github.com/DesktopECHO/Pi-hole-for-Android/tree/legacy](https://github.com/DesktopECHO/Pi-hole-for-Android/tree/legacy)
